src/lib.rs

```diff
@@ -63,6 +63,59 @@
 //! * `attiny85`
 //! * `attiny861`
 //! * `attiny88`
+//!
+//! # How to use this crate?
+//!
+//! ## Main program entry point
+//!
+//! ```
+//! #[avr_device::entry]
+//! fn main() -> ! {
+//!     loop {
+//!         // Your code here.
+//!     }
+//! }
+//! ```
```

Comment:
I think we also need to mention here that the `rt` crate feature must be active for this.